zalando-incubator/solution-center-login
# Feature/56 ignore user agent check localhost

* Code is on [Integration](https://sc-integration.norris.zalan.do)
* Refactored tests as well as added missing tests
* Refactored services to conform in `IIFE`/`$inject` style

To test (localhost):
1. `bower link` solution-center-login to SC
1. Use Chrome dev tools to emulate an iPhone.
1. Reload the page.
1. You should **not** be logged out.
1. Switch back to standard browser mode.
1. You should **not** be logged out.

To test (integration):
1. Use Chrome dev tools to emulate an iPhone.
1. Reload the page.
1. You **should** be logged out.
1. Log back in.
1. Switch back to standard browser mode.
1. You **should** be logged out. 

Closes #56
